styleguide/components/00-settings/_colors.scss

```diff
@@ -219,6 +219,10 @@ $themes: (
 
     'readspeaker-icon-color': color('cyan-wcag'),
     'readspeaker-icon-border-color': color('cyan-wcag', -2),
+
+    'tag-border-color': color('cyan-wcag', -3),
```

Comment:
Zonder rekening te houden met wcag 2.1 is er geen reden om hiervoor cyan-wcag te nemen.

Reply:
Fixed.